.azure-pipelines/generation-templates/authentication-module.yml

```diff
@@ -34,12 +34,12 @@ steps:
       - template: ../common-templates/esrp/strongname.yml
         parameters:
           FolderPath: "$(System.DefaultWorkingDirectory)/src/Authentication/Authentication"
-          Pattern: "Microsoft.Graph.Authentication.dll, Microsoft.Graph.Authentication.Core.dll"
+          Pattern: "Microsoft.Graph.Authentication.dll, Microsoft.Graph.Authentication.Core.dll, Newtonsoft.Json.dll"
 
       - template: ../common-templates/esrp/codesign.yml
         parameters:
           FolderPath: "$(System.DefaultWorkingDirectory)/src/Authentication/Authentication"
-          Pattern: "Microsoft.Graph.Authentication.dll, Microsoft.Graph.Authentication.Core.dll, *.psm1, *.psd1, *.format.ps1xml, *.ps1"
+          Pattern: "Microsoft.Graph.Authentication.dll, Microsoft.Graph.Authentication.Core.dll, Newtonsoft.Json.dll, *.psm1, *.psd1, *.format.ps1xml, *.ps1"
```

Comment:
@g2mula As the `Newtonsoft.Json.dll` isn't built by us, I'm not sure if we want to resign a 3rd party binary which we can't verify the build environment/tools as well as well as the source it was built from. (as outlined [here](https://github.com/microsoftgraph/msgraph-sdk-powershell/issues/2741#issuecomment-2123253450) by @petrhollayms, the dll is already signed and the issue looks to be trust)

I believe @fey101 is looking into re-building the package source as a submodule, so that we can sign a dll that has a verifiable chain via https://github.com/microsoftgraph/msgraph-sdk-powershell/pull/2758 to address the issue.